examples/iot-dashboard/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ if (NOT TARGET Slint::Slint)
     include(FetchContent)
 endif()
 
+include(FetchContent)
```

Comment:
Thanks! Could you just move this up to line 6 and remove the include(FetchContent) that's inside the if? :)

Reply:
Oh I didn't see it :) Sorry I will do it.